### DISCUSSION TITLE
Where can i get your Obsidian templates?

### DISCUSSION THREAD

**Discussion by Grover3293** [BryanJenksCommunity/FAQ#4] [Q&A] [answered]
Could you please provide the templates you use for literature notes and other *granular* stuff on obsidian.
Thanks

[a comment by an automated account is not shown]

**Comment by tallguyjenks** [comment, marked as answer]
@Grover3293 You can download my templates, CSS, and mermaid code after signing up for my newsletter through this link: 

[[Get My Templates and CSS]](https://bryanjenkstech.ck.page/d4ec0713d5)
